src/terraform_plan.sh

```diff
@@ -63,5 +63,6 @@ ${planOutput}
   fi
 
   echo ::set-output name=tf_actions_plan_has_changes::${planHasChanges}
+  echo ::set-output name=tf_actions_plan_output::${planOutput}
```

Comment:
I believe this needs to also follow GitHub's recommendation for multi-line strings in outputs.

```suggestion

  # https://github.community/t5/GitHub-Actions/set-output-Truncates-Multiline-Strings/m-p/38372/highlight/true#M3322
  planOutput="${planOutput//'%'/'%25'}"
  planOutput="${planOutput//$'\n'/'%0A'}"
  planOutput="${planOutput//$'\r'/'%0D'}"

  echo "::set-output name=tf_actions_plan_output::${planOutput}"
```